system_with_spring:
  ros__parameters:
    publish_freq: 10.0
    l1: 0.08
    l2: 0.2
    l3: 0.32
    lk: 0.14
    m1: 5.0
    m2: 3.0
    m3: 1.0
    a1_0: 30.0
    a2_0: 10.0
    a3_0: -30.0
    v1_0: 0.0
    v2_0: 0.0
    v3_0: 0.0
    x0: 0.1
    k: 100.0
    l0: 0.05
    T: 10.0
    h: 0.01